README.md

```diff
@@ -643,6 +643,82 @@ RerankResult result = inference.rerank(model, query, documents, rankFields, topN
 System.out.println(result.getData());
 ```
 
+# Imports
+## Start an import
+
+The following example initiates an asynchronous import of vectors from object storage into the index.
+
+```java
+import org.openapitools.db_data.client.ApiException;
+import org.openapitools.db_data.client.model.ImportErrorMode;
+import org.openapitools.db_data.client.model.StartImportResponse;
+...
+
+// Initialize pinecone object
+Pinecone pinecone = new Pinecone.Builder("PINECONE_API_KEY").build();
+// Get async imports connection object
+AsyncIndex asyncIndex = pinecone.getAsyncIndexConnection("PINECONE_INDEX_NAME");
+
+// s3 uri
+String uri = "s3://path/to/file.parquet";
+
+// Start an import
+StartImportResponse response = asyncIndex.startImport(uri, "123-456-789", ImportErrorMode.OnErrorEnum.CONTINUE);
+```
+
+## List imports
+
+The following example lists all recent and ongoing import operations for the specified index.
+
+```java
+import org.openapitools.db_data.client.ApiException;
+import org.openapitools.db_data.client.model.ListImportsResponse;
+...
+
+// Initialize pinecone object
+Pinecone pinecone = new Pinecone.Builder("PINECONE_API_KEY").build();
+// Get async imports connection object
+AsyncIndex asyncIndex = pinecone.getAsyncIndexConnection("PINECONE_INDEX_NAME");
+
+// List imports
+ListImportsResponse response = asyncIndex.listImport(100, "some-pagination-token");
```

Comment:
We need a way to call this without pagination token, since it won't be known ahead of the first call. They could pass `null` I guess, but that's kinda ugly.

Reply:
Not suggesting we do it in this PR, but seems like we could improve the user experience around pagination by seamlessly/transparently handling it within the SDK impl, and exposing via a standard iterator for example.

Reply:
> exposing via a standard iterator 

Yes, we do this in the python implementation. Seems like a good follow-up item.

Reply:
Created an internal ticket for this.